components/brave_new_tab_ui/containers/newTab/settings.tsx

```diff
@@ -219,6 +223,7 @@ export default function Settings(props: Props) {
               braveRewardsSupported={props.braveRewardsSupported}
               showRewards={props.showRewards}
             />}
+            {activeTab === TabType.Search && <SearchSettings />}
```

Comment:
I think we should update `allowedTabTypes` in L100, so that `Search` tab doesn't show up when the flag isn't enabled.